types/req-real-ip/tsconfig.json

```diff
@@ -0,0 +1,14 @@
+{
+    "compilerOptions": {
+        "module": "Node16",
```

Comment:
this does not match the source, which is commonjs.

This will also require you to use a commonjs import in the test (`import realip = require('req-real-ip')`)